src/v1/users/controllers/user-controller/index.js

```diff
@@ -1,8 +1,17 @@
+import R from 'ramda';
```

Comment:
Destructure ramda -> `import { compose } from 'ramda'`, R. (and other single letter variables, are not cool, even if they are well-known like '_' (underscore,lodash) IMO.